Api Info/## Google Places API

```diff
@@ -0,0 +1,46 @@
+## Google Places API
+ // https://developers.google.com/maps/documentation/embed/
+ -- iframe format
+
+
+
+## open weather
+  //http://openweathermap.org/api
+
+  current ,5days forecast, weather map layers
+
+EXAMPLE RESPONSE:
+
+{"city":{"id":1851632,"name":"Shuzenji",
+"coord":{"lon":138.933334,"lat":34.966671},
+"country":"JP",
+"cod":"200",
+"message":0.0045,
+"cnt":38,
+"list":[{
+        "dt":1406106000,
+        "main":{
+            "temp":298.77,
+            "temp_min":298.77,
+            "temp_max":298.774,
+            "pressure":1005.93,
+            "sea_level":1018.18,
+            "grnd_level":1005.93,
+            "humidity":87
+            "temp_kf":0.26},
+        "weather":[{"id":804,"main":"Clouds","description":"overcast clouds","icon":"04d"}],
+        "clouds":{"all":88},
+        "wind":{"speed":5.71,"deg":229.501},
+        "sys":{"pod":"d"},
+        "dt_txt":"2014-07-23 09:00:00"}
+        ]}
+
+
```

Comment:
Hi Bo!!! How Are you!!! Can you please add a full XML or JSSON structured post/get request so we can visualize ALL content that is returned to us so we know what we can use